src/components/layout/index.tsx

```diff
@@ -4,30 +4,57 @@ import CookiesBanner from "@/components/cookies";
 import { useEffect, useState } from "react";
 import Head from "next/head";
 import { urlFor } from "../../../util/client";
+import GoogleAnalytics from "../google-analytics";
+import { healpLocalStorage } from "../../../util/localStorageHelper";
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   const [isShowCookie, setIsShowCookie] = useState(true);
+  const [isConsentCookie, setIsConsentCookie] = useState(false);
   const [favicon, setFavicon] = useState("");
+  const [ga, setGa] = useState("");
+
+  const enviroment = process.env.NEXT_PUBLIC_ENVIROMENT;
   useEffect(() => {
-    const getLocalStorageCookies = localStorage.getItem("cookies");
-    const globalContent = localStorage.getItem("globalInfo") || "{}";
-    const { favicon } = JSON.parse(globalContent);
+    const getLocalStorageCookies = healpLocalStorage({
```

Comment:
should this be helpLocalStorage?
What is it doing, can we think of a name that better describes what its helping with?

Reply:
I wanted to call useLocalStorage, but use is a hook and can't be called inside useEffect, I couldn't come with a better name. That is a function for all localStorage, instead to add inline